example/tests/test_pagination.py

```diff
@@ -6,19 +6,6 @@
 pytestmark = pytest.mark.django_db
 
 
-@pytest.fixture
-def single_entry(author_factory, entry_factory):
-
-    author = author_factory(name="Joel Spolsky")
-    entry = entry_factory(
-        headline=("The Absolute Minimum Every Software Developer"
-                  "Absolutely, Positively Must Know About Unicode "
-                  "and Character Sets (No Excuses!)"),
-        blog__name='Joel on Software',
-        authors=(author, )
-    )
-
-
```

Comment:
This fixtures and the new fixture I used were moved to `conftest.py`. It's better than polluting the test files.